docs/content/patterns/review-and-submit/index.mdx

```diff
@@ -0,0 +1,305 @@
+---
+title: Review and submit
+description: Review and submit is an important step of a multi-page where users can review all information provided, make changes, understand privacy and security policies and submit the form. This pattern is used in multi-page forms.
+relatedComponents: ['progress-indicator', 'summary-list', 'task-list']
+---
+
+If there are any [declarations](/patterns/declarations), this is where they can be added.
+
+## Summary of input
+
+The layout for the ‘Review and submit’ page should include read only information from all form pages of the form Multi-page form. Each form page should have it’s own section and heading to help users scan the page.
+
+Information for each form page section should be presented to the user as a summary list. A Table can be used instead of a Summary list if a table was used on the form page. Summary lists are preferred because they provide less issues to screen reader users.
```

Comment:
```suggestion
Information for each form page section should be presented to the user as a [summary list](/components/summary-list). A [table](/components/table) can be used instead of a summary list if a table was used on the corresponding form page. Summary lists are preferred because they are easier for screen reader users to navigate.
```